### DISCUSSION TITLE
How to Auto Ack option Enable & Disable option

### DISCUSSION THREAD

**Discussion by vipin9387** [rabbitmq/rabbitmq-server#9862] [Other] [answered]
### Describe the bug

How to Auto Ack option Enable & Disable option

### Reproduction steps

How to Auto Ack option Enable & Disable option

### Expected behavior

Auto Ack option Enable & Disable option not present in Rabbitmq version 3.8.35

**Comment by michaelklishin** [comment, marked as answer]
@vipin9387 this is an extremely concise problem definition. I will guess that this is a duplicate of https://github.com/rabbitmq/rabbitmq-server/discussions/9858 (asked by you earlier today).

**Comment by michaelklishin** [comment]
@vipin9387 also, please stop repeatedly using issues for questions. That's what GitHub Discussions are for.
